GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
HPU available: False, using: 0 HPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [1]

  | Name          | Type                  | Params | Mode
----------------------------------------------------------------
0 | gw_mod        | GWModule              | 6.3 M  | train
1 | selection_mod | SingleDomainSelection | 0      | train
2 | loss_mod      | GWLosses2Domains      | 6.3 M  | train
----------------------------------------------------------------
484 K     Trainable params
5.8 M     Non-trainable params
6.3 M     Total params
25.254    Total estimated model params size (MB)
43        Modules in train mode
35        Modules in eval mode
Epoch 15:   0%|                                                                                        | 0/243 [00:00<?, ?it/s, v_num=z4in]
                                                                                                                                           
Exception ignored in: <function _releaseLock at 0x7f43de772a20>
Traceback (most recent call last):
  File "/home/alexis/miniconda3/envs/conda_no_color/lib/python3.11/logging/__init__.py", line 237, in _releaseLock
    def _releaseLock():

KeyboardInterrupt:
Exception ignored in: <function WeakValueDictionary.__init__.<locals>.remove at 0x7f42816c07c0>
Traceback (most recent call last):
  File "/home/alexis/miniconda3/envs/conda_no_color/lib/python3.11/weakref.py", line 105, in remove
    def remove(wr, selfref=ref(self), _atomic_removal=_remove_dead_weakref):

KeyboardInterrupt:

Detected KeyboardInterrupt, attempting graceful shutdown ...
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x7f4281fe3420>
Traceback (most recent call last):
  File "/home/alexis/miniconda3/envs/conda_no_color/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 1604, in __del__
    self._shutdown_workers()
  File "/home/alexis/miniconda3/envs/conda_no_color/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 1562, in _shutdown_workers
    if self._persistent_workers or self._workers_status[worker_id]:
                                   ^^^^^^^^^^^^^^^^^^^^
AttributeError: '_MultiProcessingDataLoaderIter' object has no attribute '_workers_status'
Exception ignored in atexit callback: <function _start_and_connect_service.<locals>.teardown_atexit at 0x7f41c7fac860>
Traceback (most recent call last):
  File "/home/alexis/miniconda3/envs/conda_no_color/lib/python3.11/site-packages/wandb/sdk/lib/service_connection.py", line 94, in teardown_atexit
    conn.teardown(hooks.exit_code)
  File "/home/alexis/miniconda3/envs/conda_no_color/lib/python3.11/site-packages/wandb/sdk/lib/service_connection.py", line 226, in teardown
    self._router.join()
  File "/home/alexis/miniconda3/envs/conda_no_color/lib/python3.11/site-packages/wandb/sdk/interface/router.py", line 75, in join
    self._thread.join()
  File "/home/alexis/miniconda3/envs/conda_no_color/lib/python3.11/threading.py", line 1119, in join
    self._wait_for_tstate_lock()
  File "/home/alexis/miniconda3/envs/conda_no_color/lib/python3.11/threading.py", line 1139, in _wait_for_tstate_lock
    if lock.acquire(block, timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt:
